privacy.md

```diff
@@ -346,25 +313,18 @@ of service providers that npm uses include:
 
 ### <a id="cookies">npm uses cookies.</a>
 
-npm's website uses Google Analytics to collect and analyze data about
-visitors to its websites. You can read [the privacy policy for Google
-Analytics online](https://www.google.com/analytics/terms/).
-You can opt out of Google Analytics by installing a [free browser
-extension](https://tools.google.com/dlpage/gaoptout).
-
-npm uses
-[HubSpot](https://www.hubspot.com/?__hstc=72727564.758886f1679f717bea7f57a689d3944d.1582727448385.1582727448385.1582727448385.1&__hssc=72727564.1.1582727448386&__hsfp=1312878196)
-to track which parts of npm websites you visit so we know which updates
-and service email messages to send you. We also use HubSpot to track
-when you create accounts and Orgs, when you change Orgs from free to
-paid, and when you add seats to Orgs, in order to share tips on how to
-use Orgs, to send you marketing messages about the benefits of paid
-services, and to notify our sales team that you might want paid
-services. Finally, HubSpot collects data from forms on our website that
-you must fill out to access some marketing content. You can read [the
-privacy policy for HubSpot
-online](https://legal.hubspot.com/privacy-policy?__hstc=72727564.758886f1679f717bea7f57a689d3944d.1582727448385.1582727448385.1582727448385.1&__hssc=72727564.1.1582727448386&__hsfp=1312878196).
-
+npm's website only uses cookies strictly necessary to provide, optimize
+and secure the website. For example, we use them to keep you logged in,
+remember your preferences, authenticate your device for security
+purposes, analyze your use of the service, compile statistical reports,
```

Comment:
> analyze your use of the service

Do we?

Reply:
I'm also not sure the cookies are involved in "compiling statistical reports" but I could be wrong here - cc @john7doe

Reply:
This is verbatim from the GitHub privacy policy, which npm is aligning to.